W-16815468-AddSystemProperties-24-dm (#395)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-system-properties.adoc

* Type: `Boolean`
* Default: `false`

//com.mulesoft.dw.charbuffersize//
|`com.mulesoft.dw.charbuffersize`|
Determines the size (in bytes) of the buffer used by the writer.

* Type: `Number`
* Default: `4096` 

> //mule.weave.script.parser.cache.size//

Author: ```suggestion

//mule.weave.script.parser.cache.size//
```

* Type: `Boolean`
* Default: `false`

//com.mulesoft.dw.charbuffersize//
|`com.mulesoft.dw.charbuffersize`|

> Determines the size (in bytes) of the buffer used by the writer.

Reviewer: ```suggestion
Determines the size (in bytes) of the buffer the writer uses.
```

* Type: `Number`
* Default: `4096` 

//mule.weave.script.parser.cache.size//
|`mule.weave.script.parser.cache.size`|

> Determines the maximum size of compiled scripts cached.

Author: ```suggestion
Determines the maximum size of compiled scripts that are being cached.
```

> Determines the max size of compiled scripts that are being cached.

Reviewer: ```suggestion
Determines the maximum size of compiled scripts cached.
```
